packages/contracts-bedrock/STYLE_GUIDE.md

```diff
@@ -0,0 +1,130 @@
+# Smart Contract Style Guide
+
+This document providing guidance on how we organize and write our smart contracts. For cases where
+this document does not provide guidance, please refer to existing contracts for guidance,
+with priority on the `L2OutputOracle` and `OptimismPortal`.
```

Comment:
this is the only part of the PR that could be at all controversial, but I think these contracts have had the most attention up until now and should be the final deciders if anything is unclear.